### PR TITLE
Remove no longer needed helperClassNames() methods

### PR DESCRIPTION
**Pull request by mateuszrzeszutek**
Closes https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/1374

🎉  🎉  🎉

### CODE REVIEW

File: instrumentation/aws-sdk/aws-sdk-1.11/src/main/java/io/opentelemetry/javaagent/instrumentation/awssdk/v1_11/AwsSdkInstrumentationModule.java [hunk 1]
```diff
@@ -41,6 +31,8 @@ public AwsSdkInstrumentationModule() {
 
   @Override
   public Map<String, String> contextStore() {
-    return singletonMap("com.amazonaws.AmazonWebServiceRequest", packageName + ".RequestMeta");
+    return singletonMap(
+        "com.amazonaws.AmazonWebServiceRequest",
+        "io.opentelemetry.javaagent.instrumentation.awssdk.v1_11;.RequestMeta");
```
Comment (trask) [inline instrumentation/aws-sdk/aws-sdk-1.11/src/main/java/io/opentelemetry/javaagent/instrumentation/awssdk/v1_11/AwsSdkInstrumentationModule.java:36]:
```suggestion
        "io.opentelemetry.javaagent.instrumentation.awssdk.v1_11.RequestMeta");
```

File: instrumentation/opentelemetry-api-1.0/src/main/java/io/opentelemetry/javaagent/instrumentation/opentelemetryapi/OpenTelemetryApiInstrumentationModule.java [hunk 2]
```diff
@@ -18,75 +18,6 @@ public OpenTelemetryApiInstrumentationModule() {
...
-      packageName + ".metrics.ApplicationMeter",
-      packageName + ".metrics.ApplicationMeterProvider",
-      packageName + ".trace.Bridging",
-      packageName + ".trace.Bridging$1",
-      packageName + ".trace.Bridging$2",
-      packageName + ".trace.ApplicationSpan",
-      packageName + ".trace.ApplicationSpan$Builder",
-      packageName + ".trace.ApplicationTracer",
-      packageName + ".trace.ApplicationTracerProvider",
-      packageName + ".LabelBridging"
```
Comment (trask) [inline instrumentation/opentelemetry-api-1.0/src/main/java/io/opentelemetry/javaagent/instrumentation/opentelemetryapi/OpenTelemetryApiInstrumentationModule.java:86]: 😱

**Review by trask — APPROVED** [review]
💯

File: instrumentation/aws-sdk/aws-sdk-1.11/src/main/java/io/opentelemetry/javaagent/instrumentation/awssdk/v1_11/AwsSdkInstrumentationModule.java (hunk 1 above)
Comment (mateuszrzeszutek) [reply instrumentation/aws-sdk/aws-sdk-1.11/src/main/java/io/opentelemetry/javaagent/instrumentation/awssdk/v1_11/AwsSdkInstrumentationModule.java:36]: 🤦‍♂️ 
I went too fast with that 😂

**Pull request merged**